src/org/labkey/test/components/ui/grids/ResponsiveGrid.java

```diff
@@ -180,13 +182,15 @@ protected void clickColumnMenuItem(String columnLabel, String menuText, boolean
     {
         WebElement headerCell = elementCache().getColumnHeaderCell(columnLabel);
 
-        // scroll the header cell as much to the center of the viewport as possible; if it is at the bottom the menu
-        // fly-up can be problematic to automate, if to the top sometimes that puts it behind the navbar
-        getWrapper().scrollToMiddle(headerCell);
+        // scrollIntoView handles horizontal scrolling for cases where the header is non-interactable
+        getWrapper().scrollIntoView(headerCell);    // for cells to the right or left of the viewport, scrollIntoView handles horizontal scroll
+        sleep(500);  // todo: find a way to test for whether or not x-scroll is needed, and only x-scroll if necessary
+                         //  sleep here to give scrollToMiddle call below a better chance of firing
```

Comment:
Not sure if it would work any better but have you tried scrolling the entire grid to the middle instead of the header cell? The scrolling methods might be getting confounded by the nested scrolling areas (scrolling the grid vs scrolling the whole page).
```suggestion
        // make sure grid headers are in the middle of the page
        getWrapper().scrollToMiddle(getComponentElement());
```

Reply:
I just tried your suggestion (scrolling the component, instead of the target cell) and unfortunately it wasn't successful.  I'm willing to push to TC and see if it works any better there but I doubt it will